Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'jbuilder', '~> 2.0'
 gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'devise'
 gem 'foundation-rails'
+gem 'pry-rails'
```

Comment:
The pry gem should only be loaded in the development and test environments - add it to the group at line 16!